libnmstate/plugins/nmstate_plugin_ovsdb.py

```diff
@@ -147,12 +148,30 @@ def plugin_capabilities(self):
 
     def get_interfaces(self):
         ifaces = []
-        for row in list(self._idl.tables["Interface"].rows.values()) + list(
-            self._idl.tables["Bridge"].rows.values()
-        ):
+        for row in self._idl.tables["Interface"].rows.values():
+            if row.type == "internal":
+                iface_type = InterfaceType.OVS_INTERFACE
+            elif row.type == "system":
+                # Let other plugin decide the interface type
+                iface_type = InterfaceType.UNKNOWN
+            else:
```

Comment:
Where is the `patch` interface type fits here?

Reply:
Good catch. Will shod `patch` in `ovs-interface` also.